src/modules/stateful-to-stateless.ts

```diff
@@ -278,6 +278,12 @@ export function statefulToStateless(component) {
       ) {
         copyNonLifeCycleMethods(path);
       }
+      if (t.isObjectExpression(propValue) && path.node.key.name === "state") {
+        stateHooksPresent = true;
+        propValue.properties.map(({ key, value }) => {
```

Comment:
I couldn't figure out why the linter is complaining about "key" and "value":
 `Property does not exist on type 'SpreadElement | ObjectMethod | ObjectProperty'`
But ObjectProperty does have these props and the transformation runs fine.

Reply:
Ive pushed a fix. Thats because TS has no idea that you have checked the the prop is an object expession so it treats it as a union type. Ive added casting

Reply:
Got it thanks. I'm not very familiar with TS, I'll keep an eye for that now